### ISSUE TITLE
Idea: changing some aspects of FM program

### ISSUE THREAD

**Issue by mochet** [Joystream/marketing#21]
What I am presenting here is in line with some of the goals of improving how the project can become `24/7`.

Notes: this specific idea may have implications on workload, workflow and other things, so I am more presenting it as a potential idea. Since I am not active in this part of the project, it is impossible for me to truly appreciate the potential implications of these changes, they would have to be assessed and weighed.

Current situation:
* Currently, we have the FM program divided into distinct periods of 2 weeks
* Users submit their reports, after some time grading is completed
* The grading is completed, and then submitted to a repo (https://github.com/Joystream/founding-members)
* After some period of review time/approval process the changes are merged
* The leaderboard on the website updates.
* If new users are inducted, this is announced within the Discord. Due to this not being a regular occurance, the attention this recieves is currently great and these valuable members get a good, positive experience from being inducted.
* This entire process from start to finish takes 2 weeks--meaning that only once every 2 weeks do users recieve information.

Other projects:
* Some other projects are completely dependent on Discord for their entire reputation system (and also governance)
* Depending on the project, they may have far more immediate processes for adjusting reputational information. They may also have far different standards than this project

Out of scope:
* Nothing regarding the standard of grading should be changed. This idea does not suggest anything about the standard of contributions is adjusted
* While this may suggest a change to the current workflow

Suggested changes:
* The 2 week FM period could remain and it could be clarified that reports can take up to 2 weeks to be considered/graded. This is unavoidable lag time as it ensures the high standards of the FM program are maintained.
* A Discord channel is created that exclusively "announces" changes to FM scores--we can call this the `FM scoring channel`
* Grading, when completed, is done in a way that allows for it to be updated far more immediately. This may mean using an alternative system to what GitHub currently allows or figuring out some other way of doing so.
* The changes to FM score adjustments are announced within the `FM scoring channel`
* Any specific notes regarding an individuals contributions could also be highlighted
* Information could be limited to announcing only adjustments that involve some certain significance--this might mean restricting information that is announced to those that recieve more than `400 points` in a period

Benefits:
* Surface, highlight and utilize available data in a better way
* Provide users with a somewhat continuous flow of information (due to grading not being a `24/7` process itself, it may be better to have some way of staggering the announcements)
*

**Comment by bedeho** [comment]
Thank you for this, there are other bigger changes being designed which already subsume many of these goals, but will keep this in mind.
